docs/assembly-control-unit.md

```diff
@@ -227,7 +395,7 @@ Take a moment to review and appreciate your work. This is the most fiddly part o
 - NOTE: There are 8 total places where we will be screwing wires into Phoenix connector screw terminals, each of which is an opportunity to make a mistake by changing the order of the wires. The key is to ensure that the wiring is consistent across all connections (i.e.: the leftmost wire stays the leftmost in all the junctions and cables and all other wires are also in the same order). Remember that the leftmost wire looking from the outside becomes the rightmost wire when looking from the inside.
 - We recommend ordering them alphabetically (x, Y, Z) as seen from the outside of the case.
```

Comment:
Can we capitalize this 'X' ?

Reply:
(1) Shouldn't "8 total places where we will be screwing wires" be 12? Cable has 4 wires and there are three pumps. so all 12 spots should be used right?

(2) The pump is out of stock https://www.amazon.com/gp/product/B082K6CYV1?th=1, the others have bigger housing or talk about soldering. Can we find another link or two?